hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/jdbc/SinkIntoJdbcSqlConnectorTest.java

```diff
@@ -0,0 +1,169 @@
+                        + "OPTIONS ( "
+                        + " '" + OPTION_EXTERNAL_DATASTORE_REF + "'='" + TEST_DATABASE_REF + "'"
+                        + ")"
+        );
+
+        execute("SINK INTO " + tableName + " VALUES (0, 'name-0')");
+
+        assertJdbcRowsAnyOrder(tableName, new Row(0, "name-0"));
+    }
+
+    @Test
+    public void insertIntoTableWithColumns() throws Exception {
+        createTable(tableName);
+        createMapping(tableName);
+
+        execute("SINK INTO " + tableName + " (name, id) VALUES ('name-0', 0), ('name-1', 1)");
+
+        assertJdbcRowsAnyOrder(tableName,
+                new Row(0, "name-0"),
+                new Row(1, "name-1")
+        );
+    }
+
+    @Test
+    public void insertIntoTableWithColumnsColumnHasExternalName() throws Exception {
+        createTable(tableName);
+        execute(
+                "CREATE MAPPING " + tableName + " ("
+                        + " id INT, "
+                        + " fullName VARCHAR EXTERNAL NAME name"
+                        + ") "
+                        + "TYPE " + JdbcSqlConnector.TYPE_NAME + ' '
+                        + "OPTIONS ( "
+                        + " '" + OPTION_EXTERNAL_DATASTORE_REF + "'='" + TEST_DATABASE_REF + "'"
+                        + ")"
+        );
+
+        execute("SINK INTO " + tableName + " (fullName, id) VALUES ('name-0', 0), ('name-1', 1)");
+
+        assertJdbcRowsAnyOrder(tableName,
+                new Row(0, "name-0"),
+                new Row(1, "name-1")
+        );
+    }
+
+    @Test
+    public void insertIntoTableMultipleValues() throws Exception {
+        createTable(tableName);
+        createMapping(tableName);
+
+        execute("SINK INTO " + tableName + " SELECT v,'name-' || v FROM TABLE(generate_series(0,4))");
+
+        assertJdbcRowsAnyOrder(tableName,
+                new Row(0, "name-0"),
+                new Row(1, "name-1"),
+                new Row(2, "name-2"),
+                new Row(3, "name-3"),
+                new Row(4, "name-4")
+        );
+    }
+
+
+    @Test
+    public void insertIntoTableReverseColumnOrder() throws Exception {
+        createTable(tableName, "id INT PRIMARY KEY", "name VARCHAR(10)");
+        execute(
+                "CREATE MAPPING " + tableName
+                        + " TYPE " + JdbcSqlConnector.TYPE_NAME + ' '
+                        + " OPTIONS ( "
+                        + " '" + OPTION_EXTERNAL_DATASTORE_REF + "'='" + TEST_DATABASE_REF + "'"
+                        + ")"
+        );
+
+        execute("SINK INTO " + tableName + " (name, id) VALUES ('name-0', 0)");
+
+        assertJdbcRowsAnyOrder(tableName,
+                new Row(0, "name-0")
+        );
+    }
+
+    @Test
+    public void sinkIntoTableWithColumns() throws Exception {
+        createTable(tableName);
+        createMapping(tableName);
+
+        execute("INSERT INTO " + tableName + " (name, id) VALUES ('name-0', 0), ('name-1', 1)");
```

Comment:
Ideally, we want to test just a small part of the functionality. Here a failure in INSERT will break this test, while the sink might work.

See how it's done e.g. in the test for UPDATE on line 42

https://github.com/hazelcast/hazelcast/blob/26fee11245a7d88ae5aada0ee84f30505e34e7e4/hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/jdbc/UpdateJdbcSqlConnectorTest.java#L41-L61

Reply:
Ok